.circleci/config.yml

```diff
@@ -682,8 +746,18 @@ workflows:
                 - /.*/
             tags:
               only:
-                # - /^\d+\.\d+\.\d+$/
-                # @TODO remove the following line and uncomment above after deploy testing complete
+                - /^(test\-devnet\-)*\d+\.\d+\.\d+$/
+
+      - publish_release:
+          requires:
+            - build_linux
+            - build_macos
+          filters:
+            branches:
+              ignore:
+                - /.*/
+            tags:
+              only:
                 - /^(test\-devnet\-)*\d+\.\d+\.\d+$/
 
       - build_docker_img:
```

Comment:
Below this line, `user_devnet: true` needs to be updated.

Reply:
nice catch, thanks @travisperson !